app/Http/Controllers/HomeController.php

```diff
@@ -0,0 +1,29 @@
+<?php
+namespace App\Http\Controllers;
+use Illuminate\Http\Request;
+use App\User;
+use App\Blog;
+use Illuminate\Support\Facades\Auth;
+use Carbon\Carbon;
+class HomeController extends Controller
+{
+    /**
+     * Create a new controller instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        $this->middleware('auth');
+    }
+    /**
+     * Show the application dashboard.
+     *
+     * @return \Illuminate\Contracts\Support\Renderable
+     */
+    public function index()
+    {
+        $blogs = Auth::user()->blogs()->orderBy('created_at', 'INC')->get();
+        return view('home', compact('blogs'));
```

Comment:
You did not use blogs in your home.blade.php so no need to add it here.

Reply:
Fixes issue(Delete)